Assignment-1/code/index.html

```diff
@@ -1,23 +1,34 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-<meta charset="UTF-8"/>
-<meta name="viewport" content="width=device-width">
-<title>Designing Web App using HTML/CSS/REACT</title>
-<link rel="stylesheet" href="style.css" />
-<script src="https://unpkg.com/react@17/umd/react.development.js"></script>
-<script src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
-<!-- Don't use this in production: -->
-<script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width">
+  <title>Designing Web App using HTML/CSS/REACT</title>
+  <link rel="stylesheet" href="style.css" />
+  <script src="https://unpkg.com/react@17/umd/react.development.js"></script>
+  <script src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
+  <!-- Don't use this in production: -->
+  <script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
 </head>
+
 <body>
-    <div id="root"></div>
-    <script type="text/babel">
-//React Code here
- ReactDOM.render(<div></div>,document.getElementById('root'));
+  <div id="root"></div>
+  <script type="text/babel">
+    //React Code here
+    //first div
+    var yellowDiv = React.createElement('div', { class: "yellowDiv", style: { display: "flex", 'flex-flow': "row wrap" } }, React.createElement('div', null, null), React.createElement('div', null, null))
```

Comment:
Use className instead of class while writing react codes.

Reply:
even in an HTML file or is it restricted to a js file only?

Reply:
go and check console its show class error so use className as well in html file because we writing react code here